.gitignore

```diff
@@ -43,7 +43,6 @@ scripts/run.bat
 scripts/word2md.js
 scripts/buildProtocol.js
 scripts/ior.js
-scripts/authors.js

```

Comment:
Maybe I should just leave this for a bit, until people have cleaned their script dirs enough for it to not be an annoyance.

Reply:
Nah, I have already done similar for other files. It'll be clear if some random file sneaks into a PR.